src/app/routes.tsx

```diff
@@ -1,12 +1,13 @@
-import { About, Description, Home, NotFound } from './components';
+import { About, Description, NotFound } from '@/components';
+import { Home } from '@/pages/home/home';
 
 export const routes = [
   {
     path: '/',
     element: <Home />,
     children: [
       {
-        path: '/:index',
+        path: '/:name',
```

Comment:
Дополненме к комментарию выше:

```suggestion
        path: '/pokemons/:name',
```